src/index.html

```diff
@@ -4,13 +4,62 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="GET"
+      data-qa="big"
+      class="search-form-1 search-form"
+    >
+      <div class="search-container-big search-container">
+        <label for="searchInput1">
+          <img
+            src="./images/Search.svg"
+            alt="search icon"
+            class="image-big img"
+          >
+        </label>
+        <input
+          type="text"
+          name="search"
+          id="searchInput1"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="input-big input"
+        >
+      </div>
+    </form>
+
+    <form
+      action="/"
+      method="GET"
+      class="search-form-2 search-form"
+      data-qa="small"
+    >
+      <div class="search-container-small search-container">
+        <label for="searchInput2">
```

Comment:
Names should be more descriptive, using numbers in variable names is a bad practice in most cases